e2e/autoscript-apps/src/umd-bundle-check/autoscript.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * @forgerock/javascript-sdk
+ *
+ * autoscript.ts
+ *
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+// @ts-nocheck
+import '@forgerock/javascript-sdk/bundles/index.umd.js';
+import { delay as rxDelay, map, mergeMap } from 'rxjs/operators';
+import { from } from 'rxjs';
+
+function autoscript() {
+  const delay = 0;
+
+  const url = new URL(window.location.href);
+
+  const amUrl = url.searchParams.get('amUrl') || 'https://auth.example.com:9443/am';
+  const realmPath = url.searchParams.get('realmPath') || 'root';
+  const tree = url.searchParams.get('tree') || 'Registration';
+  const un = url.searchParams.get('un') || 'f9022889-4452-48a0-aa94-182436645551';
+  const pw = url.searchParams.get('pw') || 'password';
+  const email = url.searchParams.get('email') || 'sally.tester@me.com';
+
+  console.log('Configure the SDK');
+  try {
+    forgerock.Config.set({
+      realmPath,
+      tree,
+      serverConfig: {
+        baseUrl: amUrl,
+      },
```

Comment:
I believe all of these properties are optional, so you should be able to successfully call `set` with an empty object and call it a day.